src/Kiota.Builder/Refiners/DartRefiner.cs

```diff
@@ -289,6 +289,7 @@ private static void CorrectPropertyType(CodeProperty currentProperty)
         else if (currentProperty.IsOfKind(CodePropertyKind.BackingStore))
         {
             currentProperty.Type.Name = currentProperty.Type.Name[1..]; // removing the "I"
+            currentProperty.SerializationName = currentProperty.Name;
```

Comment:
Why are we normalizing serialization names here? It should be the original value we received from the document.

Reply:
The serialization name is not normalized. Specifically, this line _preserves_ the original name in the document, so it is correctly serialized after normalizing the property name.

Reply:
sorry, I reviewed this PR too fast and half awake. disregard my confused self